packages/core/src/gui/MaxWindow.ts

```diff
@@ -235,7 +235,7 @@ class MaxWindow extends EventSource {
     y: number,
     width: number | null = null,
     height: number | null = null,
-    style = 'MaxWindow'
+    style = 'mxWindow'
```

Comment:
**thought:** the common.css file should be review to only defined mxWindow rules. Currently, there is a mix of mxWindow and MaxWindow rules.
This should be possible to validate this by calling MaxLog.show